### ISSUE TITLE
testing on small data set

### ISSUE THREAD

**Issue by junkyul** [junkyul/allenel#6]
run C model; 
check model;
learn vocab;
now don't use pre-trained glove embedding.

how long?
entity dense coherence dense vs.
sparse sparse

type model

predictor from learned mode? evaluate?

**Comment by junkyul** [comment]
current run encounters cuda memory out

word embedding -- pretrained :
passes 300 dim vector per word

if not pre-traiend:
reader passes list of token ids 
and the embedding matrix is Vocab size x 300


inside model; free?

reduce batch size to 100

**Comment by junkyul** [comment]
remove comments; forward variables

all embeddings sparse (sentence, entity, cohernce)
densh_sparse_adam
comment out type related things

**Comment by junkyul** [comment]
1.5 sec / 1000 batch iteration
using the first file, accuracy Conll ~ 0.9? No
